src/OrchardCore.Modules/OrchardCore.ContentLocalization/Drivers/ContentCulturePickerSettingsDriver.cs

```diff
@@ -48,7 +49,7 @@ public override async Task<IDisplayResult> UpdateAsync(ContentCulturePickerSetti
                 return null;
             }
 
-            if (context.GroupId == GroupId)
+            if (context.GroupId.Equals(GroupId, StringComparison.OrdinalIgnoreCase))
```

Comment:
you may want to create an extension to the `BuildShapeContext` class that can be reused when comparing group?

something like this

```
public static class BuildShapeContextExtensions
{
	public static class SameGroupAs(this BuildShapeContext context, string groupId)
	{
		return context.GroupId.Equals(groupId, StringComparison.OrdinalIgnoreCase);
	}
}
```

Reply:
I don't see any value here to create an extension with a single line code, but let me revise the PR again and see what we could come up with